strategies/SimplePatienceStrategyV2.py

```diff
@@ -0,0 +1,32 @@
+from .IterativeStrategy import IterativeStrategy
+
+
+class SimplePatienceStrategyV2(IterativeStrategy):
```

Comment:
I had structured patience strategies to inherit from each other: poly > linear > simple. So, with the change to `b` for patience in this class (mentioned in another comment), we won't need to redefine the patience function anymore.